examples/workflows/hello_world.py

```diff
@@ -6,5 +6,9 @@ def hello(s: str):
     print("Hello, {s}!".format(s=s))
 
 
-with Workflow(generate_name="task-exit-handler-", entrypoint="s") as w:
-    hello(s="hello")
+with Workflow(
+    generate_name="hello-world-",
+    entrypoint="hello",
+    arguments={"s": "world"},
```

Comment:
Should `arguments={"s": "world"},` go to `hello`?

Reply:
They do go to `hello` - this example is actually the same as the upstream `arguments_parameters` [here](https://hera.readthedocs.io/en/stable/examples/workflows/upstream/arguments_parameters/) except using a script!